growth/too/models.py

```diff
@@ -145,8 +145,12 @@ def create_all():
             reference_images = {}
             reference_mags = {}
 
-        tessfile = pkg_resources.resource_stream(__name__,
-                                                 'input/%s.tess' % tele)
+        tessfile = os.path.join(app.instance_path, '%s.tess' % tele)
+        if os.path.isfile(tessfile):
```

Comment:
Instead of checking if the file exists, please open it in a `try` block and as a fallback open the bundled copy in the `except` block.